cmd/kyma/alpha/deploy/localdomain.go

```diff
@@ -0,0 +1,72 @@
+package deploy
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+)
+
+// LocalKymaDevDomain contains the default domain name if Kyma is locally deployed
+const LocalKymaDevDomain = "local.kyma.dev"
+
+// KymaLocalDomain configures the local kyma domain in Kubernetes DNS
+type KymaLocalDomain struct {
+	coreDNSConfig []byte
+	kubeClient    kubernetes.Interface
+}
+
+// New creates a new Kyma local domain instance
```

Comment:
Could you please document what's exactly happening here?

Reply:
Good point, as it's a not self-explaining change. Added more documentation to it.